cmd/dcrdata/views/extras.tmpl

```diff
@@ -2,11 +2,17 @@
 <head data-turbolinks-eval="false">
 	<meta charset="utf-8">
 	<meta name="viewport" content="width=device-width, initial-scale=1">
-	<!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
+	<!-- The above meta tags *must* come first in the head -->
 	<meta name="description" content="dcrdata, an original Decred block explorer powered by Go">
 	<meta name="author" content="The Decred developers">
-	<meta name="application-name"           content="Decred - Autonomous Digital Currency">
-	<meta name="apple-mobile-web-app-title" content="Decred - Autonomous Digital Currency">
+	<meta name="application-name"           content="dcrdata - The Decred Block Explorer">
+	<meta name="apple-mobile-web-app-title" content="dcrdata - The Decred Block Explorer">
+
+	<meta name="twitter:card" content="summary"/>
```

Comment:
OpenGraph tags are also needed to enable preview cards on non-Twitter websites (eg. Facebook). 

```
<meta property="og:title"        content="" >
<meta property="og:type"         content="" />
<meta property="og:url"          content="" />
<meta property="og:image"        content="" />
<meta property="og:description"  content="" />
```

Reply:
Thanks @jholdstock.  Was going to look into that next.  

One thing that I noticed on https://developer.twitter.com/en/docs/twitter-for-websites/cards/overview/abouts-cards, is that they recommend pretty huge sizes for the `summary_large_image` card type, so I went with summary and just a reasonably sized symbol image.  The docs there also say not to use static images, so presumably we'd want to move toward a slightly more dynamic image depending on the page. e.g. Supplement the logo image with a little "block" drawing for a /block page, some arrows for the /tx page, maybe a postage stamp or for /address.  Not planning to do this though.

Reply:
I wouldn't invest too much time into getting this "perfect", I only opened #1822 to make sure we have **something** instead of absolutely nothing.

Making sure we have at least an image and a bit of text is the key thing. If that can be dynamic depending on the page type, thats great, but optimising for every possible resolution/pixel-density isn't so important IMO.